conf/docker/dataverse-glassfish/Dockerfile

```diff
@@ -57,7 +59,63 @@ RUN chgrp -R 0 /usr/local/glassfish4 && \
+
+#JHOVE
+RUN cp /tmp/jhove* /usr/local/glassfish4/glassfish/domains/domain1/config
+
+
+#SETUP JVM OPTIONS 
+RUN cd /tmp \
+    && cd /usr/local/glassfish4/glassfish/bin \
+    && ./asadmin start-domain \
+    &&./asadmin $ASADMIN_OPTS delete-jvm-options "-XX\:MaxPermSize=192m" \
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "-XX\:MaxPermSize=512m"\
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "-XX\:PermSize=256m"\
+    && ./asadmin $ASADMIN_OPTS delete-jvm-options -client \
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.files.directory=/usr/local/glassfish4/glassfish/domains/domain1/files"\
+    # Rserve-related JVM options: 
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.rserve.host=localhost" \
+     && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.rserve.port=6311" \
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.rserve.user=rserve" \
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.rserve.password=rserve" \
+    # Data Deposit API options
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.fqdn='localhost:8080'" \
+    # password reset token timeout in minutes
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.auth.password-reset-timeout-in-minutes=60"\
+
+    # EZID DOI Settings
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddoi.password=apitest" \
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddoi.username=apitest" \
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddoi.baseurlstring=https\://ezid.cdlib.org"  \
+    # "I am the timer server" option:
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "-Ddataverse.timerServer=true" \
+
+# enable comet support
+    && ./asadmin $ASADMIN_OPTS set server-config.network-config.protocols.protocol.http-listener-1.http.comet-support-enabled="true" \
+
+    && ./asadmin $ASADMIN_OPTS delete-connector-connection-pool --cascade=true jms/__defaultConnectionFactory-Connection-Pool \
+    && ./asadmin $ASADMIN_OPTS create-connector-connection-pool --steadypoolsize \
+1 --maxpoolsize 250 --poolresize 2 --maxwait 60000 --raname jmsra \
+--connectiondefinition javax.jms.QueueConnectionFactory jms/IngestQueueConnectionFactoryPool \
+
+# http://docs.oracle.com/cd/E18930_01/html/821-2416/abllx.html#giogt
+    && ./asadmin $ASADMIN_OPTS create-connector-resource --poolname \
+jms/IngestQueueConnectionFactoryPool --description "ingest connector resource" jms/IngestQueueConnectionFactory \
+
+# http://docs.oracle.com/cd/E18930_01/html/821-2416/ablmc.html#giolr
+    && ./asadmin $ASADMIN_OPTS create-admin-object --restype \
+javax.jms.Queue --raname jmsra --description "sample administered object" --property Name=DataverseIngest jms/DataverseIngest \
+# so we can front with apache httpd ( ProxyPass / ajp://localhost:8009/ )
+    && ./asadmin $ASADMIN_OPTS create-network-listener --protocol http-listener-1 --listenerport 8009 --jkenabled true jk-connector \
```

Comment:
We don't want to maintain all these `asadmin` commands both here and in `scripts/installer/glassfish-setup.sh` do we? Can we have the Dockerfile call `glassfish-setup.sh` instead?